src/main/java/com/epam/tc/hw4/pages/EpamInformationFrameworkPage.java

```diff
@@ -0,0 +1,205 @@
+    private WebElement elementContactsOnNavigationBar;
+
+    @FindBy(linkText = "SERVICE")
+    private WebElement elementServiceOnNavigationBar;
+
+    @FindBy(linkText = "METALS & COLORS")
+    private WebElement elementMetalsAndColorsOnNavigationBar;
+
+    @FindBy (css = "li.dropdown:nth-child(1) > a:nth-child(1)")
+    private WebElement dropdownButtonWhereLoginFunctions;
+
+    @FindBy(xpath = "//span[text() = 'Home']")
+    private WebElement homeElementOnLeftSideMenu;
+
+    @FindBy(xpath = "//span[text() = 'Service']")
+    private WebElement serviceElementOnLeftSideMenu;
+
+    @FindBy(xpath = "//span[text() = 'Contact form']")
+    private WebElement contactsElementOnLeftSideMenu;
+
+    @FindBy(xpath = "//span[text() = 'Metals & Colors']")
+    private WebElement metalsAndColorsElementOnLeftSideMenu;
+
+    @FindBy(xpath = "//span[text() = 'Elements packs']")
+    private WebElement elementsPacksElementOnLeftSideMenu;
+
+    @FindBy(xpath = "//span[text() = 'Different elements']")
+    private WebElement differentPage;
+
+    @FindBy(id = "name")
+    private WebElement userNameFieldForLogin;
+
+    @FindBy(id = "password")
+    private WebElement passwordFieldForLogin;
+
+    @FindBy(id = "login-button")
+    private WebElement buttonForLogin;
+
+    @FindBy(id = "user-name")
+    private WebElement loginUserName;
+
+    @FindBy(className = "icon-base")
+    private WebElement iconBase;
+
+    @FindBy(className = "icon-practise")
+    private WebElement iconPractise;
+
+    @FindBy(className = "icon-multi")
+    private WebElement iconMulti;
+
+    @FindBy(className = "icon-custom")
+    private WebElement iconCustom;
+
+    @FindBy(xpath = "//span[starts-with(text(),'To include good practices')]")
+    private WebElement textUnderPractiseIcon;
+
+    @FindBy(xpath = "//span[starts-with(text(),'To be flexible')]")
+    private WebElement textUnderCustomIcon;
+
+    @FindBy(xpath = "//span[starts-with(text(),'To be multiplatform')]")
+    private WebElement textUnderMultiIcon;
+
+    @FindBy(xpath = "//span[starts-with(text(),'Already have good')]")
+    private WebElement textUnderBaseIcon;
+
+    @FindBy(id = "frame")
+    private WebElement iframe;
+
+    @FindBy(id = "frame-button")
+    private WebElement iframeButton;
+
+    @Step ("Login to account")
+    public void login(String password, String username) {
+        dropdownButtonWhereLoginFunctions.click();
+        userNameFieldForLogin.sendKeys(username);
+        passwordFieldForLogin.sendKeys(password);
+        buttonForLogin.click();
+    }
+
+    @Step ("Verify that user have proper username after login")
+    public void verifyLoginUser(String usernameToVerify) {
+        softAssertions.assertThat(usernameToVerify).isEqualTo(loginUserName.getText());
+    }
+
+    @Step ("Verify that image with name 'Base' is displayed on the page")
+    public void verifyThatBaseIconIsDisplayed() {
+        softAssertions.assertThat(iconBase.isDisplayed()).isTrue();
```

Comment:
I don't think isTrue() is not needed here
This comment also true for other isTrue() calls